test/test_account.py

```diff
@@ -342,6 +348,7 @@ async def test_invalid_password_china():
         mock_api.post("/eadrax-coas/v2/login/pwd").respond(
             422, json=load_response(RESPONSE_DIR / "auth" / "auth_cn_login_error.json")
         )
+        mock_api.post("/eadrax-coas/v1/cop/check-captcha").respond(422)
```

Comment:
Please also test if check fails first and then succeeds on a later request.

Reply:
I am a Python beginner and I'm trying to see how to complete this test.

Reply:
For being a beginner, this is very, very good!

Please find an example of a first error message and then success here: https://github.com/bimmerconnected/bimmer_connected/blob/master/test/test_account.py#L192